lib/Rex/User/Linux.pm

```diff
@@ -158,6 +158,15 @@ sub create_user {
     }
   }
 
+  # don't lock new users
+  if ( ! (
+       exists $data->{crypt_password}
+    || exists $data->{password}
+    || exists $data->{system} ) ) {
```

Comment:
Can you elaborate why we're checking for system here? This means system users will still be created as locked no? Maybe it's a good idea to mention this in the comment section above.

Reply:
We are checking for system to not alter the current behavior for the system user creation.
Also system users should not be allowed to login eg to run the login shell.